requirements/dev-requirements.txt

```diff
@@ -13,6 +13,7 @@ alembic==0.8.6
 amqp==1.4.9
 anyjson==0.3.3
 appdirs==1.4.3
+appnope==0.1.0            # via ipython
```

Comment:
confused by this line. https://github.com/dimagi/commcare-hq/pull/21563

not sure if it's necessary?

Reply:
"The Internet" says that appnope is a dependency of ipython on OSX, but not Linux.  Probably https://github.com/dimagi/commcare-hq/pull/21563 was generated in a Linux environment?

Reply:
https://github.com/jazzband/pip-tools/issues/277

Reply:
This also changes prod-requirements.txt. Since our servers run linux, I think I'd default to whatever is generated.

With that said, from the linked ticket, it doesn't seem like there's a way to enforce this consistently. I'll create an issue and ping danny on it, to see if he knows more after he gets back from offline time